build.sbt

```diff
@@ -811,7 +812,9 @@ lazy val runtime = (project in file("engine/runtime"))
         "org.graalvm.truffle" % "truffle-api"           % graalVersion      % Benchmark,
         "org.typelevel"      %% "cats-core"             % catsVersion,
         "eu.timepit"         %% "refined"               % refinedVersion
-      ),
+    ),
+    GenerateAST.rustVersion := rustVersion,
+    Compile / sourceGenerators += GenerateAST.task,
```

Comment:
You need to update the contributing guide section on [rust setup](https://github.com/enso-org/enso/blob/main/docs/CONTRIBUTING.md#getting-set-up-rust) to mention that SBT will check the rust version. You should also add a link to rustup as the recommended way to install and manage toolchains. Not sure how I forgot that before.